tools/scripts/build/templateInstallationPackage/autonomiccsJars/moveJars.sh

```diff
@@ -23,10 +23,24 @@
 # Autonomiccs jars
 cp autonomiccsJars/autonomic*.jar /usr/share/cloudstack-management/webapps/client/WEB-INF/lib/
 cp autonomiccsJars/starthost-plugin*.jar /usr/share/cloudstack-management/webapps/client/WEB-INF/lib/
-# Do not change this line, we remove the '.jar' extension on purpose.
-wakeOnLanNewFileName=$(ls autonomiccsJars/wakeonlan-service-*.jar) && wakeOnLanNewFileName=${wakeOnLanNewFileName##*/} && wakeOnLanNewFileName=${wakeOnLanNewFileName::-4}
-cp autonomiccsJars/wakeonlan-service-*.jar "/usr/share/cloudstack-management/webapps/client/WEB-INF/lib/$wakeOnLanNewFileName";
 
 # Autonomiccs platform dependencies.
 cp autonomiccsJars/dependencies/*.jar /usr/share/cloudstack-management/webapps/client/WEB-INF/lib/
 
+# Creation of a folder to hold the jar files that needs to get transfered to VMs.
+baseAutonomiccsJarFolder="/var/lib/autonomiccs/jars/";
+
+jadeAgentsFolder="${baseAutonomiccsJarFolder}jade/";
+wakeOnLanFolder="${baseAutonomiccsJarFolder}wakeonlan/";
+
+wakeOnLanFileName="wakeonlan-service";
+jadeAgentsFileName="jade-plataform-agents";
+
+jadeAgentsRegex="autonomiccsJars/${jadeAgentsFileName}-*.jar";
```

Comment:
you do not need to use ${jadeAgentsFileName}, you can use simply $jadeAgentsFileName.
The same happens for the next lines